src/model/CplacePlugin.ts

```diff
@@ -58,7 +59,7 @@ export default class CplacePlugin {
         if (fs.existsSync(e2ePath)) {
             this.hasTypeScriptE2EAssets = glob.sync(path.join(e2ePath, '**', '*.ts')).length > 0;
         }
-
+        this.hasYamlAssets = glob.sync(path.join(this.pluginDir, 'api', '*.yaml')).length > 0;
```

Comment:
The name "yaml assets" seems a bit misleading, because we are not dealing with assets in the sense of the other assets (js, css, and less). Maybe we should talk about "yaml api" instead?